spyder/plugins/projects/widgets/main_widget.py

```diff
@@ -168,8 +169,12 @@ def __init__(self, name, plugin, parent=None):
         self.treewidget.sig_open_file_requested.connect(
             self.sig_open_file_requested)
 
-        # Empty widget to show when there's no active project
-        self.emptywidget = ProjectExplorerTreeWidget(self)
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "projects",
+            _("No project opened yet"),
```

Comment:
```suggestion
            _("No project opened"),
```

Since the user could, of course, close their existing project at any time (and perhaps later open another, "yet" doesn't make sense here. Therefore, we can fix this (and make it shorter/simpler) by simply deleting it.